### PR TITLE
Add Keras integration to documentation

### PR DESCRIPTION
**Pull request by merveenoyan**
This PR adds Keras docs to documentation.

### CODE REVIEW

File: docs/hub/keras.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+model = from_pretrained_keras("keras-io/mobile-vit-xxs")
+prediction = model.predict(image)
+prediction = tf.squeeze(tf.round(prediction))
+print(f'The image is a {classes[(np.argmax(prediction))]}!')
+
+# The image is a sunflower!
+```
+
+
+If you want to see how to load a specific model, you can click `Use in keras` and you will be given a working snippet that you can load it!
```
Comment (NimaBoscarino) [inline docs/hub/keras.md:36]:
```suggestion
If you want to see how to load a specific model, you can click **Use in keras** and you will be given a working snippet that you can load it! 
```

File: docs/hub/keras.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+If you want to see how to load a specific model, you can click `Use in keras` and you will be given a working snippet that you can load it! 
+
+![snippet](docs/assets/hub/keras_snippet1.png)
+![snippet](docs/assets/hub/keras_snippet2.png)
+
+## Sharing your models
+
+You can share your `keras` models by using the `push_to_hub_keras` method. This will generate a model card that includes your model’s hyperparameters, plot of your model and couple of sections related to the usage purpose of your model, biases the training data might have and limitations about putting your model to production, puts the metrics of your model in a JSON if model has a history.
```
Comment (NimaBoscarino) [inline docs/hub/keras.md:43]: The second sentence in this paragraph is really long and ends up being a bit confusing. Is it possible to split it into two or three?

File: docs/hub/keras.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+from huggingface_hub import push_to_hub_keras
+
+push_to_hub_keras(model,
+    "your-username/your-model-name",
+    "your-tensorboard-log-directory",
+    tags = ["object-detection", "some_other_tag"],
+    **model_save_kwargs,
+)
+```
+The repository will host your tensorboard traces like below.
```
Comment (NimaBoscarino) [inline docs/hub/keras.md:55]:
```suggestion
The repository will host your TensorBoard traces like below.
```

File: docs/hub/keras.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+---
+title: Keras
+---
+
+# Using Keras at Hugging Face
+
+`Keras` is an open-source machine learning library that leverages PyTorch and Python to provide high-level components to train neural networks with state-of-the-art outputs on all modalities.
```
[the remark this answers is not in the document]
Comment (merveenoyan) [reply docs/hub/keras.md:7]: hehe I kinda copied the fastai one sorry lol let me add :D

File: docs/hub/keras.md (hunk 2 above)
Comment (merveenoyan) [reply docs/hub/keras.md:43]: sure!

File: docs/hub/keras.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+## Using existing models
+
+The `huggingface_hub` library is a lightweight Python client with utlity functions to download models from the Hub.
+
+```
+pip install huggingface_hub["tensorflow"]
+```
+
+Once you have the library installed, you just need to use the `from_pretrained_keras` method.
```
[the remark this answers is not in the document]
Comment (merveenoyan) [reply docs/hub/keras.md]: Addressed!

File: docs/hub/keras.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+title: Keras
+---
+
+# Using Keras at Hugging Face
+
+`Keras` is an open-source machine learning library that leverages TensorFlow and Python to provide high-level components to train neural networks with state-of-the-art outputs on all modalities.
+
+## Exploring Keras in the Hub
+
+You can find `Keras` models by iltering at the left of the [models page](https://huggingface.co/models?library=keras&sort=downloads). All models on the Hub come up with useful features such as an automatically generated model card and metadata tags that help for discoverability. If provided by the model owner, TensorBoard logs are hosted on the Keras repositories as well.
```
Comment (NimaBoscarino) [inline docs/hub/keras.md:11]:
```suggestion
You can find `Keras` models by filtering at the left of the [models page](https://huggingface.co/models?library=keras&sort=downloads). All models on the Hub come up with useful features such as an automatically generated model card and metadata tags that help for discoverability. If provided by the model owner, TensorBoard logs are hosted on the Keras repositories as well.
```

File: docs/hub/keras.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+
+## Using existing models
+
+The `huggingface_hub` library is a lightweight Python client with utlity functions to download models from the Hub.
```
Comment (NimaBoscarino) [inline docs/hub/keras.md:16]:
```suggestion
The `huggingface_hub` library is a lightweight Python client with utility functions to download models from the Hub.
```

**Review by NimaBoscarino — APPROVED** [review]

**Review by NimaBoscarino — APPROVED** [review]

**Comment by osanseviero** [comment]
Thanks for this PR!

**Pull request merged**